core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -63,6 +65,12 @@ static RdbEngineStrategy create(String jdbcUrl) {
 
   protected abstract String getDataTypeForEngine(DataType dataType);
 
+  protected abstract String getDataTypeForKey(DataType dataType);
```

Comment:
This class has mixed `protected` and `package private` methods. Is there a reason for this?

Reply:
Just because I didn't clearly distinguished the difference between `protected` and `package private` methods when I first started the refactoring in https://github.com/scalar-labs/scalardb/pull/787. (I'm a Java newbie)

Now I feel using `package private` or `public` would be the best if the `RdbEngineStragety` stays `abstract class`.
(Append to L68 is a `protected` method maybe because it's a similar method as L66 and I copy-and-pasted it)

NOTE: `RdbEngineStrategy` will become `interface` in https://github.com/scalar-labs/scalardb/pull/791 so all the methods will be `public`.

Reply:
> NOTE: RdbEngineStrategy will become interface in https://github.com/scalar-labs/scalardb/pull/791 so all the methods will be public.

Okay. Then, it's fine for now. Thanks.